build/config/BUILDCONFIG.gn

```diff
@@ -245,7 +245,7 @@ if (target_os == "android") {
 } else if (target_os == "chromeos" || target_os == "linux") {
   # See comments in build/toolchain/cros/BUILD.gn about board compiles.
   if (is_clang) {
-    _default_toolchain = "//build/toolchain/linux:clang_$target_cpu"
+    _default_toolchain = "//cobalt/build/toolchain/linux:cobalt_clang_$target_cpu"
```

Comment:
This looks like an all or nothing change for all build configs. I'm assuming it's not possible to gate it here with `if cobalt`, to leave regular Chromium configs untouched ?

Reply:
We definitely can do an `is_cobalt` check and should—I believe Niranjan is working on this, we talked about it last Friday

Reply:
my bad, forgot this. ill add this

Reply:
done